### PR TITLE
gobuster 1.3 (new formula)

### PR DESCRIPTION
**Pull request by leonjza**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gobuster.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/OJ").mkpath
+    ln_sf buildpath, buildpath/"src/github.com/OJ/gobuster"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", bin/"gobuster"
+  end
+
+  test do
+    system "#{bin}/gobuster"
```
Comment (woodruffw) [inline Formula/gobuster.rb]: Might be good to test the output of `gobuster` here, instead of just running it.

Comment (leonjza) [inline Formula/gobuster.rb]: Sure thing! Will update the test to something like `assert_match(/\[!\] WordList \(-w\): Must be specified/, shell_output("#{bin}/gobuster -q"))` later tonight.

**Comment by leonjza** [comment]
Test added!

**Comment by ilovezfs** [comment]
Who ya gonna call?

**Comment by dunn** [comment]
Thanks! Merges as da67d2f89ee0348222818698352344a0825cda85 👻

**Pull request closed without merging**